### PR TITLE
MGDCTRS-1625: Elasticsearch sink connector fails with wrong body type exception (tests)

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Comment by rinaldodev** [comment]
@lburgazzoli can you return the Ignore to azure eventhubs? They are still failing eventually.

**Comment by lburgazzoli** [comment]
> @lburgazzoli can you return the Ignore to azure eventhubs? They are still failing eventually.

done

**Comment by lburgazzoli** [comment]
@rinaldodev ok to merge ?

File: cos-fleet-catalog-connectors-it/nosql/elasticsearch-it/src/test/groovy/org/bf2/cos/connector/camel/it/elasticsearch/ElasticsearchConnectorIT.groovy [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            ]
+
+            if (user != null && password != null) {
+                props['elasticsearch_user'] = user
+                props['elasticsearch_password'] = password
+            }
+
+            def cnt = connectorContainer('elasticsearch_sink_0.1.json', props)
+
+            cnt.withUserProperty('quarkus.log.category."org.apache.camel.component.elasticsearch".level', 'DEBUG')
```
Comment (rinaldodev) [inline cos-fleet-catalog-connectors-it/nosql/elasticsearch-it/src/test/groovy/org/bf2/cos/connector/camel/it/elasticsearch/ElasticsearchConnectorIT.groovy:49]: We can now use: https://github.com/bf2fc6cc711aee1a0c2a/cos-fleet-catalog-camel/blob/8404805044bad4b1019193c684330562f798b888/cos-fleet-catalog-connectors-it-support/src/main/java/org/bf2/cos/connector/camel/it/support/ConnectorContainer.java#L184-L187

**Review by rinaldodev — APPROVED** [review]

**Pull request merged**